CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# Unreleased
+
+## Added
+
+- Implemented `Eq` for `common::headers::Encoding`, `common::headers::MediaType`, 
```

Comment:
Nit: I think it would be better to include the full path e.g. `std::ops::Eq`

Reply:
`Eq` is in the prelude and so prevalent in rust that I do not think this clarification is needed